frontend/src/features/admin-form/settings/components/MrfFormEmailSection.tsx

```diff
@@ -0,0 +1,205 @@
+import { useCallback, useMemo } from 'react'
+import { Controller, useForm } from 'react-hook-form'
+import { Box, FormControl, FormErrorMessage, Skeleton } from '@chakra-ui/react'
+import { debounce, get, isEmpty, isEqual, uniq } from 'lodash'
+import isEmail from 'validator/lib/isEmail'
+
+import { MultirespondentFormSettings } from '~shared/types/form'
+
+import { OPTIONAL_ADMIN_EMAIL_VALIDATION_RULES } from '~utils/formValidation'
+import { MultiSelect } from '~components/Dropdown'
+import FormLabel from '~components/FormControl/FormLabel'
+import { TagInput } from '~components/TagInput'
+
+import { useAdminFormWorkflow } from '~features/admin-form/create/workflow/hooks/useAdminFormWorkflow'
+
+import { useMutateFormSettings } from '../mutations'
+
+interface MrfEmailNotificationsFormProps {
+  settings: MultirespondentFormSettings
+  isDisabled: boolean
+}
+
+const WorkflowEmailMultiSelectName = 'email-multi-select'
+const OtherPartiesEmailInputName = 'other-parties-email-input'
+
+const MrfEmailNotificationsForm = ({
+  settings,
+  isDisabled,
+}: MrfEmailNotificationsFormProps) => {
+  const { isLoading, formWorkflow } = useAdminFormWorkflow()
+
+  const formWorkflowStepsWithStepNumber =
+    formWorkflow?.map((step, index) => ({
+      ...step,
+      stepNumber: index + 1,
+    })) ?? []
+
+  const checkIsEmail = useMemo(() => isEmail, [])
```

Comment:
This doesn't seem to be doing anything special other than to reassign to a different variable name? 🤔